### PR TITLE
123

### PR DESCRIPTION
**Pull request by iluxa-com**

### CODE REVIEW

**Comment by iluxa-com** [comment]
sadf

File: Quicksort.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    quicksort(0, number - 1);
+  }
+
+  private void quicksort(int low, int high) {
+    int i = low, j = high;
+    // Get the pivot element from the middle of the list
+    int pivot = numbers[low + (high-low)/2];
+
+   
+
```
Comment (iluxa-com) [inline Quicksort.java:24]: sdaf

File: Quicksort.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
+//This is quick sort
+
+public class Quicksort  {
+  private int[] numbers;
+  private int number;
```
Comment (iluxa-c0m) [inline Quicksort.java:5]: 5

File: Quicksort.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+  public void sort(int[] values) {
+    // Check for empty or null array
+    if (values ==null || values.length==0){
+      return;
```
Comment (iluxa-c0m) [inline Quicksort.java:10]: left pane comment

File: Quicksort.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
+//This is quick sort
```
Comment (iluxa-c0m) [inline Quicksort.java:1]: | `line 13` |
| --- |

left apne 13

**Comment by iluxa-c0m** [comment]
sadfdsa

File: Quicksort.java (hunk 4 above)
Comment (iluxa-c0m) [inline Quicksort.java:1]: | `line 18` |
| --- |

ads

File: Quicksort.java [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+   
+
+      // If we have found a values in the left list which is larger then
+      // the pivot element and if we have found a value in the right list
+      // which is smaller then the pivot element then we exchange the
+      // values.
+      // As we are done we can increase i and j
+      if (i <= j) {
+        swap(i, j);
+        i++;
```
Comment (iluxa-c0m) [inline Quicksort.java:32]: asdf

**Pull request closed without merging**